sale_blanket_order/views/sale_blanket_order_views.xml

```diff
@@ -214,17 +205,13 @@
                                         domain="[('type_tax_use','=','sale')]"
                                         context="{'default_type_tax_use': 'sale'}"
                                         options="{'no_create': True}"
-                                        attrs="{
-                                            'required': [('display_type', '=', False)],
-                                            'invisible': [('display_type', '=', True)],
-                                        }"
+                                        required="display_type == False"
+                                        invisible="display_type == True"
```

Comment:
Please use `column_invisible` see https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0